apps/dashboard/app/views/files/_shell_dropdown.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="shell-wrapper btn-group dropright">
+  <%= link_to OodAppkit.shell.url(path: @path.to_s).to_s, id: 'open-in-terminal-btn', class: 'btn btn-outline-dark btn-sm disabled', target: '_blank' do %>
```

Comment:
Seems like we don't need to disable this button anymore since the partial can be rendered during the initial load AND during navigation via AJAX